styles/utils/_variables.scss

```
@@ -0,0 +1,3 @@
+$main-font-color: #292929;
```

Good practice is to name color variable like:
```suggestion
$corol-grey: #292929;
```